Update content/en/docs/Validators/_index.md (#26)
chainx-org/documentation: content/en/docs/Validators/_index.md

## Synchronize Chain Data

Run the command below to synchronize chain data.

```bash
$ ./chainx --chain=mainnet --pruning=archive
```

When completed, you need to restart the node with the validator flag.

```bash
$ ./chainx --chain=mainnet --validator
```

Or you can run the above command directly because the validator node will run with `archive` by default.

> IMPORTANT! You should wait for node syncing completely and set the session key correctly before you as a candidate.

Reviewer: ```suggestion
IMPORTANT! You should wait for the completion of node syncing and ensure the session key is set correctly before you become a candidate.
```

> {{%alert%}}If the error occurred when node syncing, you need to check if the system time consists of the internet time and remove the full database before you try again.{{%/alert%}}

Reviewer: ```suggestion
{{%alert%}}If the error occurred when node syncing, you need to check if the system time is consistent with the internet time and remove the full database before you try again.{{%/alert%}}
```

}
```

> {{%alert color="warning"%}} Some RPC is dangerous for validator node. If you want to expose, please use a proxy server to filter. Visit [https://github.com/paritytech/substrate/wiki/Public-RPC](https://github.com/paritytech/substrate/wiki/Public-RPC) for details. If you are aware of and understand the potential risks, you can enable it by add `--rpc-method unsafe` to starting arguments.{{%/alert%}}

Reviewer: ```suggestion
{{%alert color="warning"%}} Some RPCs are dangerous for the validator node. If you want to expose them, please use a proxy server to filter. Visit [https://github.com/paritytech/substrate/wiki/Public-RPC](https://github.com/paritytech/substrate/wiki/Public-RPC) for details. If you are aware of and understand the potential risks, you can enable it by adding the argument `--rpc-method unsafe`.{{%/alert%}}
```
